redash/query_runner/databend.py

```diff
@@ -0,0 +1,118 @@
+try:
+    from databend_sqlalchemy import connector
+    import re
+
+    enabled = True
+except ImportError:
+    enabled = False
+
+from redash.query_runner import BaseQueryRunner, register
+from redash.query_runner import TYPE_STRING, TYPE_INTEGER, TYPE_BOOLEAN, TYPE_FLOAT, TYPE_DATETIME, TYPE_DATE
+from redash.utils import json_dumps, json_loads
+
+
+class Databend(BaseQueryRunner):
+    noop_query = "SELECT 1"
+
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "host": {"type": "string", "default": "localhost"},
+                "port": {"type": "int", "default": 8000},
+                "username": {"type": "string", "default": "root"},
```

Comment:
Looks like it's required and doesn't really have a default value?

Reply:
Fixed, thanks!